chia/farmer/farmer.py

```diff
@@ -670,6 +674,36 @@ def get_receiver(self, node_id: bytes32) -> Receiver:
             raise KeyError(f"Receiver missing for {node_id}")
         return receiver
 
+    def check_missing_signage_points(
```

Comment:
is there no way to test this new function?

Reply:
I'll add test case.